### PR TITLE
Move to DotNetCore-Build pool

### PR DESCRIPTION
**Pull request by karajas**
This pool includes VS2017.

/cc @eerhardt

### CODE REVIEW

**Comment by eerhardt** [comment]
Let's not merge this until we get a clean official build.  We've been broken for a few days.

**Comment by eerhardt** [comment]
Removing "no merge" as we now have a green official build.

File: buildpipeline/security/DotNet-Core-Setup-Security-Windows.json [hunk 1]
```diff
@@ -560,11 +560,11 @@
         "imageUrl": "https://devdiv.visualstudio.com/_api/_common/identityImage?id=9d5fdf9f-36b6-4d0c-a12e-2737a673af94"
         },
     "queue": {
-        "id": 36,
-        "name": "DotNet-Build",
+        "id": 330,
```
Comment (eerhardt) [inline buildpipeline/security/DotNet-Core-Setup-Security-Windows.json]: corefx and coreclr don't use this pool.  Why do we?

/cc @mmitche

Comment (mmitche) [reply buildpipeline/security/DotNet-Core-Setup-Security-Windows.json]: @eerhardt They should be soon.

**Pull request merged**